samples/LoggingTracer/LoggingTracer/LoggingTracerFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace LoggingTracer
+{
+    using OpenTelemetry.Context;
+    using OpenTelemetry.Context.Propagation;
+    using OpenTelemetry.Trace;
+
+    public class LoggingTracerFactory : ITracerFactory
+    {
+        public ITracer Create(string name)
+        {
+            Logger.Log($"TracerFactory.Create({name})");
+            // Here
+            // - return new Tracer
+            // - or return Tracer from a map for the the given name.
+            // - or return (new) Tracer based on a condition (e..g config) otherwise NopTracer, ... 
+            // - ...
+            return new LoggingTracer();
```

Comment:
We could pass the tracer name to the LoggingTracer instance. It could use the name as logging prefix.